azure-pipelines.vrt-pr.yml

```diff
@@ -27,53 +27,32 @@ jobs:
 
       - template: .devops/templates/tools.yml
 
-      - bash: |
-          postPolicy="true";
-          response=$(curl --request POST ${VR_APP_CLIENT_URL} --header 'Content-Type: application/x-www-form-urlencoded' --data-urlencode 'grant_type=client_credentials' --data-urlencode 'client_id='${VR_APP_CLIENT_ID} --data-urlencode 'client_secret='${VR_APPROVAL_CLIENT_SECRET})
-            parsedResponse=${response/*"access_token"/}
-            token=${parsedResponse:3:${#parsedResponse}-5}
-            curl --location --request POST 'https://vrtfunctionappv0.azurewebsites.net/api/policyStateV2' \
-            --header 'Authorization: Bearer '"$token" \
-            --header 'Content-Type: application/json' \
-            --data-raw '	{
-            "organization": "uifabric",
-            "projectName": "fabricpublic",
-            "prId": $(System.PullRequest.PullRequestNumber),
-            "commitId": "$(Build.SourceVersion)",
-              "generate":true,
-              "clientType":"FLUENTUI",
-              "blockingPipeline":{
-              },
-              "nonBlockingPipeline":{
-                "$(pipelineId)": {
-                  "pipelineStatus": "PENDING",
-                  "pipelineName": "$(pipelineName)"
-                }
-              },
-              "postPolicy": '${postPolicy}',
-              "policyType": "OPTIONAL"
-            }'
-        displayName: 'Call policy State Api'
-        env:
-          VR_APPROVAL_HOST: $(VR_APPROVAL_HOST)
-          VR_APP_CLIENT_ID: $(VR_APP_CLIENT_ID)
-          VR_APPROVAL_CLIENT_SECRET: $(VR-APPROVAL-CLIENT-SECRET)
-          VR_APP_CLIENT_URL: $(VR_APP_CLIENT_URL)
-
       - template: .devops/templates/runpublishvrscreenshot.yml
         parameters:
           fluentVersion: webcomponents
           vrTestPackageName: '@fluentui/vr-tests-web-components'
           vrTestPackagePath: 'apps/vr-tests-web-components'
 
+      - bash: |
```

Comment:
**not part of this PR -> follow up:** can we remove most of the repetition by actually moving all these steps under yml template  ( move everything to ` template: .devops/templates/runpublishvrscreenshot.yml` etc )